static/app/views/insights/common/components/tableCells/starredTransactionCell.tsx

```diff
@@ -0,0 +1,77 @@
+import {useState} from 'react';
+
+import {Button} from 'sentry/components/core/button';
+import {IconStar} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {FlexContainer} from 'sentry/utils/discover/styles';
+import useProjects from 'sentry/utils/useProjects';
+import {
+  type StarTransactionParams,
+  useStarredTransactions,
+} from 'sentry/views/insights/common/utils/useStarredTransactions';
+
+interface Props {
+  containerProps?: React.DetailedHTMLProps<
+    React.HTMLAttributes<HTMLDivElement>,
+    HTMLDivElement
+  >;
+  isStarred?: boolean;
+  projectSlug?: string;
+  transactionName?: string;
+}
+
+export function StarredTransactionCell({
+  transactionName,
+  isStarred: initialIsStarred,
```

Comment:
I like this rename a lot! I actually think it'd be good if the prop was _called_ `initialIsStarred` to make it clear that this component is not controlled

Reply:
That's a good idea!